premio_cms/templates/child_specific.html

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
rename this template to "location_preview.html".  "child specific" doesnt really mean anything, also rename "child_arg" to "page"